modelconverter/packages/base_exporter.py

```diff
@@ -39,7 +44,8 @@ def __init__(
         self.disable_onnx_simplification = config.disable_onnx_simplification
         self.disable_onnx_optimization = config.disable_onnx_optimization
 
-        self.model_name = self.input_model.stem
+        self.model_name = sanitize_net_name(input_model.stem)
+        self.original_model_name = input_model.name
```

Comment:
```suggestion
        self.original_model_name = sanitize_net_name(
            input_model.name, with_suffix=True
        )
```

That is so the final file moved from the intermediate_outputs has the correct name with the correct suffix (following the suggestion on the `sanitize_net_name` method.